plugins/uast-kotlin/src/org/jetbrains/uast/kotlin/KotlinUastLanguagePlugin.kt

```diff
@@ -298,7 +196,8 @@ internal object KotlinConverter {
 
     internal fun convertPsiElement(element: PsiElement?,
                                    givenParent: UElement?,
-                                   requiredType: Class<out UElement>?): UElement? {
+                                   requiredType: Array<out Class<out UElement>>
```

Comment:
I feel like it would be a good idea to rename this to something like `allowedTypes`.
Not critical, of course.